server/lib/fetch.py

```diff
@@ -400,6 +400,45 @@ def property_values(nodes, prop, out=True, constraints=''):
   return result
 
 
+def multiple_property_values(nodes: List[str],
+                             props: List[str],
+                             out=True) -> Dict[str, Dict[str, List[str]]]:
+  """
+  Fetches specified properties for given nodes using the /v2/node API.
+
+  Args:
+      nodes (List[str]): List of node identifiers.
```

Comment:
should wereplace "identifiers" by dcid just to be more specific?

Reply:
fixed